crates/proof-of-sql/src/base/mod.rs

```diff
@@ -1,5 +1,6 @@
 //! This module contains basic shared functionalities of the library.
 /// TODO: add docs
+pub mod arrow;
```

Comment:
```rust
#[cfg(feature = "arrow")]
```
Please put it above `pub mod arrow` and remove all instances of it within the `arrow` directory